stories/layout/Page.stories.js

```diff
@@ -28,14 +28,17 @@ const secondaryActions = [
       </span>
     ),
     onClick: action('Save Clicked'),
+    as: 'button',
```

Comment:
Looks like this will need to be added to the app as well?

Reply:
yep, it defaults to `link` which is the same as the old implementation